src/main.rs

```diff
@@ -18,9 +18,15 @@ impl ZellijPlugin for State {
         let session_name = pipe_message.payload.unwrap().to_string();
         let collection: Vec<&str> = session_name.split("::").collect::<Vec<&str>>().clone();
         let session_name = collection[0];
-        let cwd = Some(PathBuf::from(collection[1]));
         let layout_name = "default";
         let layout: LayoutInfo = LayoutInfo::File(layout_name.to_string());
+        if collection.len() < 2 {
+            switch_session_with_layout(Some(&session_name), layout, None);
+            close_self();
+            return true;
+        }
+
+        let cwd = Some(PathBuf::from(collection[1]));
```

Comment:
IMO it's better to do it like this:

```
let mut cwd = None;
if collection.len() == 2 {
    cwd = Some(PathBuf::from(collection[1]));
}
```

Reply:
addressed the requested changes in [9fdba4d](https://github.com/mostafaqanbaryan/zellij-switch/pull/1/commits/9fdba4d91faa58cb983f480e62c70444e3c820ce)